### PR TITLE
Updated data file used for Getting Started and Profiling notebooks (#3262)

### PR DESCRIPTION
**Pull request by ckrapu**
This is my first pull request on Github and addresses issue #3262 . The "Getting Started" Jupyter notebook uses Pandas datareader to download approximately 400 days' worth of daily financial returns and then fits the stochastic volatility model to that data. A nearly identical file is already saved for use in the "Profiling" notebook, though that file does not have the dates for each data point and does not have a NaN value in the first row as is the case for the data obtained with datareader. I replaced the existing SP500.csv file with one that has a NaN value in the first row removed and also has the date for each data point included. Both notebooks now use that file and there have been minor modifications to the code in each one to reflect this. I have rerun both notebooks and confirm that they give the desired output. 

In short, I modified an existing data file we already use so that it can be used in both notebooks and we no longer rely on datareader for either of them. I would appreciate any feedback regarding how I have made the changes or described them.

### CODE REVIEW

**Comment by twiecki** [comment]
This is great and I agree with the approach taken.

- [ ] `pd.read_csv(get_data('SP500.csv'),index_col=0)` add space after comma and set `parse_dates=True`)
- [ ] `returns = pd.read_csv(pm.get_data('SP500.csv'),parse_dates=['date'],index_col=0)` same note as above
- [ ] Add note in release-notes.

**Comment by twiecki** [comment]
Actually while you're at it, can you also change `conda install -c conda-forge pymc3` to `conda install pymc3` and `via conda-forge` to just `via conda` in the getting started?

**Comment by ckrapu** [comment]
I change the notebooks per your recommendation and added a line in the release notes under "maintenance". There are extra commits here because 1) I made a commit with a file that shouldn't have been included and had to revert and 2) I did not update my repo with the updates made since last night and thus had to resolve a merge conflict in the release notes.

**Comment by twiecki** [comment]
Thanks @ckrapu !
